vmdb/app/models/orchestration_stack.rb

```diff
@@ -8,7 +8,7 @@ class OrchestrationStack < ActiveRecord::Base
 
   has_ancestry
 
-  belongs_to :ext_management_system, :foreign_key => :ems_id, :class_name => "EmsCloud"
+  belongs_to :ext_management_system, :foreign_key => :ems_id, :class_name => "ExtManagementSystem"
```

Comment:
@Fryguy / @lfu, does this do anything that might break reporting?  I don't fully grok how the `class_name` option impacts the reporting engine when defining relationships.